util/eventbus/fetch-htmx.go

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) Tailscale Inc & AUTHORS
+// SPDX-License-Identifier: BSD-3-Clause
+
+//go:build ignore
+
+package main
```

Comment:
We should add a brief doc comment in here IMO.
```suggestion

// Program fetch-htmx fetches and installs local copies of the HTMX library
// and its dependencies, used by the debug UI. It is meant to be run via
// go generate.
package main
```

or words to that effect maybe.

Reply:
Added!